yakuake-init.sh

```diff
@@ -25,24 +25,55 @@ trim ()
     echo $var
 }
 
+# debugging helper function
+#
+# usage:
+# logm "There is an ERROR o.O!"
+#
+# view log:
+# $ tail -f /var/log/syslog
+#
+logm ()
+{
+    logger $(trim ${1})
+}
+
 # Helper that sends a command to yakuake
 call_yakuake_method ()
 {
     path="${1}"
     method="${2}"
     parameters=("${@:3}")
-    result=$(dbus-send --print-reply=literal --dest=org.kde.yakuake /yakuake/${path} org.kde.yakuake.${method} "${parameters[@]}")
-    result=$(trim ${result})
 
-    if [[ -n "${result}" ]]
+    if [[ -z "${parameters}" ]]
     then
-        echo "${result}"
+	    result=$(dbus-send --print-reply=literal --dest=org.kde.yakuake /yakuake/${path} org.kde.yakuake.${method})
```

Comment:
We seem to have a difference of opinion here about whitespace ;). Please stick with the already established convention of 4 spaces per indentation level throughout the file.